ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -591,6 +591,25 @@ msgstr "Récemment créé"
 msgid "Datasets with data"
 msgstr "Jeux de données avec données"
 
+#: ckanext/ontario_theme/templates/snippets/search_form.html:29
+msgid "Keywords"
+msgstr "Mots clés"
+
+#: ckanext/ontario_theme/templates/snippets/search_form.html:29
+msgid "Formats"
+msgstr "Formats"
+
+#: ckanext/ontario_theme/templates/snippets/search_form.html:29
+msgid "Licenses"
+msgstr "Licences"
+
+#: ckanext/ontario_theme/templates/snippets/search_form.html:29
+msgid "Access Level"
```

Comment:
Missing removal of access level and frequency from POT file.  Handled in merge conflicts.